docs/_templates/index.html

```diff
@@ -217,21 +217,128 @@ <h2 class="featurette-heading">One log, many analyses. <span class="text-muted">
                         in Memray - there are many more to choose from, allowing
                         you to view your data in the way that makes the most
                         sense for your specific needs.
-                    </p>.
+                    </p>
                 </div>
                 <div class="col-md-5">
                     <img class="featurette-image img-fluid mx-auto reveal_l"
                         src="{{ pathto('_static/images/stats_example.png', 1) }}" alt="Many reporters">
                 </div>
             </div>
 
-            <hr class="featurette-divider">
-
+            <div class="row featurette">
+                <h2 class="featurette-heading">Memray in the media</h2>
+                <div class="row">
+                    <br>
+                    <div class="card-columns">
+                        <a href="https://news.ycombinator.com/item?id=31102089" class="card">
+                            <div class="card-body">
+                              <h5 class="card-title">Memray: a memory profiler for Python</h5>
+                              <h6 class="card-subtitle mb-2 text-muted">Hacker News</h6>
```

Comment:
Should we include the date on this (April 20th, 2022)? Every other entry has one.